frontend/src/types/pipeline/task.ts

```diff
@@ -51,6 +54,21 @@ export type TaskDatabaseSchemaUpdatePayload = {
   pushEvent?: VCSPushEvent;
 };
 
+export type TaskDatabaseSchemaUpdateGhostPayload =
+  TaskDatabaseSchemaUpdatePayload & {
+    // empty by now
```

Comment:
Added a new field `SocketFileName`

Reply:
We might need not to (better don't) expose SocketFileName to frontend.